tremor-script/src/registry.rs

```diff
@@ -15,7 +15,7 @@
 mod custom_fn;
 pub use self::custom_fn::CustomFn;
 pub(crate) use self::custom_fn::{RECUR_PTR, RECUR_REF};
-use crate::ast::{BaseExpr, NodeMetas};
+use crate::ast::{AstEq, BaseExpr, NodeMetas};
```

Comment:
same as with the visitor, can we pull AstEq in it's own module to shrink the at file a bit?